cardano-config/src/Cardano/Config/Shelley/Genesis.hs

```diff
@@ -14,42 +14,25 @@ module Cardano.Config.Shelley.Genesis
 
 import           Cardano.Prelude
 
-import qualified Data.Text.Encoding as Text
 import qualified Data.Map.Strict as Map
 import qualified Data.Time as Time
 
-import           Data.Aeson (Value, ToJSON(..), toJSON, (.=),
-                             FromJSON(..), (.:),
-                             ToJSONKey(..),   ToJSONKeyFunction(..),
-                             FromJSONKey(..), FromJSONKeyFunction(..))
+import           Data.Aeson (Value, ToJSON(..), toJSON,
+                             FromJSON(..))
 import           Data.Aeson.Types    (Parser)
-import qualified Data.Aeson          as Aeson
-import qualified Data.Aeson.Encoding as Aeson
-import qualified Data.ByteString.Base16 as Base16
 
-import           Control.Monad.Fail (fail)
-
-import           Cardano.Crypto.Hash.Class as Crypto
+import           Cardano.Config.Orphanage ()
```

Comment:
Could we just keep the genesis orphan instance here in this module and rename the module something like `Cardano.Config.Shelley.GenesisOrphans`? It'd keep the module structure much clearer. I don't like us mixing the Byron and Shelley things unnecessarily.

Reply:
Yep that makes sense, I'll make the change.